wsgi/odyssy/odyssy/settings.py

```diff
@@ -114,3 +114,17 @@
 # https://docs.djangoproject.com/en/1.8/howto/static-files/
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(WSGI_DIR, 'static')
+
+
+# This will allow cookies  to be sent only via HTTPS connections.
+SESSION_COOKIE_SECURE = True
+CSRF_COOKIE_SECURE = True
+
+
+# Terminates the session as soon as browser closes.
+SESSION_EXPIRE_AT_BROWSER_CLOSE = True
```

Comment:
I agree, but considering the use case of ours (a college website) we don't need this kind of security. Using persistent sessions will make our website more user friendly. I think that most of our users will not be using this website on cyber cafe and nowadays most of the people has personal smartphones and laptops. 
If you still want more security we can send an email to the user every time there is a login from a new device.